packages/framework/src/DataFixtures/Demo/BrandDataFixture.php

```diff
@@ -65,6 +65,14 @@ public function load(ObjectManager $manager)
                 'cs' => 'Toto je popis značky ' . $brandData->name . '.',
                 'en' => 'This is description of brand ' . $brandData->name . '.',
             ];
+
+            $brandData->seoTitles[1] = null;
+            $brandData->seoTitles[2] = null;
+            $brandData->seoMetaDescriptions[1] = null;
+            $brandData->seoMetaDescriptions[2] = null;
+            $brandData->seoH1s[1] = null;
+            $brandData->seoH1s[2] = null;
+
```

Comment:
Demo data fixtures are executed before second domain is created, you should add the nulls only for Domain::FIRST_DOMAIN_ID and add the second domain in DemoMultidomain (or let it be automatically created during create-demo-data phing target)

Reply:
I created DemoDomainFixture. I think that it belongs into create-demo-data but there is plan which will change behaviour of this target, so I will leave it to implementation of that US